### DISCUSSION TITLE
Credit card deduction failed for supabase pro

### DISCUSSION THREAD

**Discussion by AlbertGithubHome** [supabase/supabase#29101] [Questions] [answered]
This morning I received an email from supabase alerting me to a failed credit card charge. Because my credit card has reached its limit, I am looking for another credit card to replace it. I want to know **how long** this bill is not paid, will suspend my service, I don't want my service to be affected. Please understand the friends to give answers, thank you very much.

**Comment by GaryAustin1** [comment]
You need to contact support.

**Comment by AlbertGithubHome** [comment, marked as answer]
roughly two weeks yeah.
